docs/book/src/tasks/verify-container-images.md

```diff
@@ -0,0 +1,54 @@
+# Verification of CAPI artifacts
+
+## Requirements 
+You will need to have the following tools installed:
+- cosign [(install guide)](https://docs.sigstore.dev/system_config/installation/)
+- jq [(download jq)](https://stedolan.github.io/jq/download/)
+
+## CAPI Images
+Each [release](https://github.com/kubernetes-sigs/cluster-api/releases) of the Cluster API project includes the following container images:
+- cluster-api-controller
+- kubeadm-bootstrap-controller
+- kubeadm-control-plane-controller
+- clusterctl
```

Comment:
We also reference the dev images in the release manifests (which are hosted on the gcr staging registry):
- capd-manager
- capim-manager
- test-extension

But I guess the target group here are the users who only require the non-dev images. So should be fine to not include them here?!

Reply:
Hey @johannesfrey Thanks for the review. Appreciate that. As you've mentioned already, it's more targeted to end users, so I think the latter approach of not including dev images sounds better. I'm open to more feedback and thoughts on this.

Reply:
Sounds fine for me :slightly_smiling_face: